### PR TITLE
Landing page done!

### PR DESCRIPTION
**Pull request by BradleyLi**
The landing page is finished except the footer. Please review my push request. Thanks!

### CODE REVIEW

File: server/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
+{
```
Comment (cloudyyoung) [inline server/tsconfig.json:1]: This file should be gone.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (cloudyyoung) [inline package.json:1]: There shouldn't be a package.json file in the root directory (when mono-repo causes confusion 😅)

File: postcss.config.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (cloudyyoung) [inline postcss.config.js:1]: Same.

File: package-lock.json [hunk 4]
```diff
@@ -0,0 +1,1551 @@
+{
```
Comment (cloudyyoung) [inline package-lock.json:1]: Same.

File: tailwind.config.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
```
Comment (cloudyyoung) [inline tailwind.config.js:1]: Same.

File: client/tailwind.config.js [hunk 6]
```diff
@@ -2,7 +2,22 @@
...
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        'gray': '#D9D9D9',
+        'pink': '#FF2480',
+        'getStarted': '#B7194A',
+        'learnMore': '#B7194A',
+        'divider': "ADADAD",
+        'start': '#B7194A'
```
Comment (cloudyyoung) [inline client/tailwind.config.js:12]: This is not how extended color palette should be used (and class naming should be kabab-case)

**Review by cloudyyoung — CHANGES_REQUESTED** [review]
Further improvements are to be posted after #18 is merged :)

**Comment by tim-macphail** [comment]
It works!

**Review by tim-macphail — APPROVED** [review]
It's not perfect but that's fine, we can make it better in later PRs 😄

**Review by jaza-k — APPROVED** [review]

**Pull request merged**